jinja/carnival/prize_booth.html

```diff
@@ -0,0 +1,79 @@
+{% extends 'base.html' %}
+{% block content %}
+{% block navbar %}
+{% include "navbar.html" %}
+{% endblock %}
+
+<div class="parallax" style="background-image: url('../img/xanadu-background.png'); box-shadow: none;">
+    <div class="container">
+
+        <div class="mt-5 pt-5 text-center teal-text text-lighten-2">
+            <img src="../img/carnival_logo_splash.png" class="logo"></img>
+        </div>
+        <div class="row-sm text-center justify-content-center" style="color:grey; font-family: 'Roboto', sans-serif !important;">
+            <h3 class="col-12 mt-5 mb-5 grey-text">
+                Cash your points to win our amazing <strong>Carnival Prizes</strong>. 
+            </h3>
+            
+	<div style="font-size: 14pt;">
+            <p>
+            The participants with the most points will be the winners of some cool Xanadu swag and more!
+
+</p>
+
+<p>
+Make sure to save all your ticket codes to cash your PennyLane #QuantumCarnival prizes! 🎟️
+
+</p>
+
+<p>
+Here's a sneak-peek of all the cool things you can get for your points!
+<br>
+</p>
+
+        </div>
+        </div>
+        <div class = "row align-items-center">    
+            <div class="col">
+                <img src="../img/xanadu_logo_hoodie.png" class="logo" style="max-width: 300px;"></img>
+            </div>
+            <div class="col">
+                <img src="../img/xanadu_t_shirt.jpeg" class="logo" style="max-width: 300px;"></img>
+            </div>
+            <div class="col">
+                <img src="../img/xanadu_backpack.png" class="logo" style="max-width: 300px;"></img>
+
+            </div>
+        </div>
+        <div class = "row justify-content-center align-items-center">    
+            <div class="col">
+                <img src="../img/xanadu_bottle_opener.jpg" class="logo" style="max-width: 300px;"></img>
+            </div>
+            <div class="col">
+                <img src="../img/xanadu_baseball_hat.jpg" class="logo" style="max-width: 300px;"></img>
+            </div>
+            <div class="col">
+                <img src="../img/xanadu_notebook.png" class="logo" style="max-width: 200px;"></img>

```

Comment:
Hmm, this one looks "fake". Better not to include this image than to have it I'd say